include/openspace/scene/translation.h

```diff
@@ -46,7 +46,16 @@ class Translation : public properties::PropertyOwner {
 
     glm::dvec3 position(double time);
 
+    // Registers a callback that gets called when a significant change has been made that
+    // invalidates potentially stored points, for example in trails
+    void onParameterChange(std::function<void()> callback);
```

Comment:
I know we have a lot of these void onXYZ(function<void()>) methods already, and they are nice and create a simple API. But as soon as we want an observable have more than 1 observer, it becomes a bit more tricky. I don't think this is a problem here at all, but it would be good to have a discussion about these kind of pub-sub issues at some point :)